Project.toml

```
@@ -55,7 +55,7 @@ IsApprox = "0.1"
 KrylovKit = "0.4.2, 0.5, 0.6, 0.7"
 LinearAlgebra = "1.6"
 LinearMaps = "3"
-NDTensors = "0.3.0"
+NDTensors = "0.3.0, 0.4"
```

```suggestion
NDTensors = "0.3.0"
```
if we just bump the patch version.